### PR TITLE
Release v2.0.4 rc1

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

**Comment by bitcartel** [comment]
So this RC release PR does not include a message about the release height or when EOS halt will kick in.  We can do this for the full release.

Do we want to add the message in the doc folder, release notes for v2.0.4?
The main readme.md?
Or the Github release notes and the release Blog Post? (I prefer this option, keeps it out of the existing release process)

File: doc/authors.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ Cory Fields (35)
 mdr0id (22)
 Larry Ruane (16)
 syd (15)
+Jonathan "Duke" Leto (15)
 Matt Corallo (13)
-Jonathan "Duke" Leto (12)
+Ariel Gabizon (11)
```
Comment (daira) [inline doc/authors.md]: Similarly, `Ariel Gabizon` now has 11 commits, when previously `Ariel Gabizon` and `Ariel` collectively had 12 commits. Off-by-one error?

File: doc/release-notes/release-notes-2.0.4-rc1.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Miners who wish to use the same address for every mined block, should use the
+`-mineraddress` option. 
+
+[Mining Guide](https://zcash.readthedocs.io/en/latest/rtd_pages/zcash_mining_guide.html)
+
+
+New consensus rule: Reject blocks that violate turnstile (Testnet only)
+-----------------------------------------------------------------------
+Testnet nodes will now enforce a consensus rule which marks blocks as invalid
+if they would lead to a turnstile violation in the Sprout or Shielded value
```
Comment (daira) [inline doc/release-notes/release-notes-2.0.4-rc1.md]: Sprout or Sapling.

Comment (bitcartel) [reply doc/release-notes/release-notes-2.0.4-rc1.md]: Strange, I force pushed a change to PR https://github.com/zcash/zcash/pull/3900 but not picked up.  I will fix in this PR.

File: doc/authors.md (hunk 1 above)
Comment (bitcartel) [reply doc/authors.md]: Don't know, automated script handles this.

File: doc/authors.md [hunk 3]
```diff
@@ -17,12 +17,12 @@ Cory Fields (35)
 mdr0id (22)
 Larry Ruane (16)
 syd (15)
+Jonathan "Duke" Leto (15)
```
[the remark this answers is not in the document]
Comment (bitcartel) [reply doc/authors.md]: Don't know, automated script handles this.

File: doc/release-notes/release-notes-2.0.4-rc1.md (hunk 2 above)
Comment (bitcartel) [reply doc/release-notes/release-notes-2.0.4-rc1.md]: @daira Pushed commit

**Comment by bitcartel** [comment]
@zkbot r+

**Comment by bitcartel** [comment]
@zkbot treeclosed=1

**Pull request merged**